back-end/e-learning/app/Http/Controllers/WordController.php

```diff
@@ -59,4 +59,51 @@ public function list($id)
 		
 		return response($collection, 201);
 	}
+
+	public function update(Request $request, $id)
+	{
+		$collection = new Collection();
+
+		$word = Word::find($id);
+		$word->value = $request->word;
+		$word->save();
+
+		$choices = Choice::where('word_id', $id)->get();
```

Comment:
or you can just do `$choices = $word->choices` or `$choices = $word->choices()->get()`